cmd/peek-submission/main.go

```diff
@@ -21,7 +16,7 @@ var args struct {
 	CourseID         string `help:"ID of the course." arg:""`
 	AssignmentID     string `help:"ID of the assignment." arg:""`
 	TargetSubmission string `help:"ID of the submission. Defaults to the latest submission." arg:"" optional:""`
-	Short            bool   `help:"Use short form output."`
+	Verbose          bool   `help:"Use verbose output to show full request/response without specific formatting." default:"false"`
```

Comment:
I think we need to be a little creative here.

Note that this option is really owned by internal/cmd,
since it is the one that actually defines the behavior.
I think we can do something similar to config.ConfigArgs here and make a struct that contains all the base CMD arguments (as of now it is only verbose).
Then this args struct can just embed the common CMD args struct and pass those args along to the cmd functions (see how we pass args.ConfigArgs).